modules/develop/partials/escape-hatch.adoc

```diff
@@ -2,6 +2,7 @@ This
 ifdef::page-topic-type[{page-topic-type}]
 ifndef::page-topic-type[page]
 is for Couchbase Capella.
-// tag::link[]
-For Couchbase Server, see xref:server:develop:{docname}.adoc[].
-// end::link[]
\ No newline at end of file
+ifdef::flag-escape-hatch[]
+// hide-escape-hatch enables authors to override the escape hatch for individual pages
```

Comment:
ooo, cute

Reply:
But unnecessary 😞 — if you [soft set](https://docs.antora.org/antora/latest/playbook/asciidoc-attributes/#soft-set) the `flag-devex-escape-hatch` flag when building the site, authors can override the escape hatch by unsetting the flag for individual pages.